### PR TITLE
Add client-go 7.0.0 changelog

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://docs.google.com/spreadsheets/d/14A9bijhCBHju5V0R4leKaa-ugcm-pEC9SZ0zJFHpVBk/edit#gid=0

/cc @sttts @caesarxuchao @timoreimann @munnerz @LiliC

### CODE REVIEW

**Comment by nikhita** [comment]
@sttts Wanted to double check: having a PR authoritatively merged in client-go won't create problems with the publishing bot, will it?

**Comment by caesarxuchao** [comment]
I'll take a look today. Thank you @nikhita.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,119 @@ https://github.com/kubernetes/client-go/issues/234.
...
+
+**Breaking Changes:**
+
+* Google Cloud Service Account email addresses can now be used in RBAC Role bindings since the default scopes now include the `userinfo.email` scope. This is a breaking change if the numeric uniqueIDs of the Google service accounts were being used in RBAC role bindings. The behavior can be overridden by explicitly specifying the scope values as comma-separated string in the `users[*].config.scopes` field in the `KUBECONFIG` file.
+
+    * [https://github.com/kubernetes/kubernetes/pull/58141](https://github.com/kubernetes/kubernetes/pull/58141)
+
+* [k8s.io/api] `VolumeScheduling` and `LocalPersistentVolume` features are beta and enabled by default. Action Required: The PersistentVolume NodeAffinity alpha annotation is deprecated and will be removed in a future release.
+
+    * [https://github.com/kubernetes/kubernetes/pull/59391](https://github.com/kubernetes/kubernetes/pull/59391)
```
[the remark this answers is not in the document]
Comment (nikhita) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,119 @@ https://github.com/kubernetes/client-go/issues/234.
...
+
+* A `/token` subresource for ServiceAccount is added.
+
+    * [https://github.com/kubernetes/kubernetes/pull/58111](https://github.com/kubernetes/kubernetes/pull/58111)
+
+* If an informer delivery fails, the particular notification is skipped and continued the next time.
+
+    * [https://github.com/kubernetes/kubernetes/pull/58394](https://github.com/kubernetes/kubernetes/pull/58394)
+
+* When using client or server certificate rotation, the Kubelet will no longer wait until the initial rotation succeeds or fails before starting static pods. This makes running self-hosted masters with rotation more predictable.
```
[the remark this answers is not in the document]
Comment (nikhita) [reply CHANGELOG.md]: Done. Thanks for the suggestion, @caesarxuchao!

**Comment by sttts** [comment]
> having a PR authoritatively merged in client-go won't create problems with the publishing bot, will it?

We keep the changelog still here in client-go (at least until we have a proper release sub-process as part of the big release process of kube).

**Comment by sttts** [comment]
@nikhita can you create a follow-up for the README?

**Comment by nikhita** [comment]
> @nikhita can you create a follow-up for the README?

Forgot to do it here. Will do!

**Pull request merged**